docs/content/dagster-cloud/account/managing-users.mdx

```diff
@@ -1,668 +1,90 @@
-width={2064}
-height={1052}
-/>
-
-To remove an override, click **Edit user role** next to the code location to open the user role selection dialog. Then, click the **Remove override** button:
+<Note>
+  <strong>Organization Admin</strong> permissions are required to manage users
+  in Dagster Cloud.
+</Note>
 
-<!-- ![Removing the override for a code location](/images/dagster-cloud/user-token-management/remove-code-location-override.png) -->
+After a user is created, the **Manage user permissions** window will automatically display. You can also access this window by clicking **Edit** next to a user in the users table.
 
 <Image
-alt="Removing the override for a code location"
-src="/images/dagster-cloud/user-token-management/remove-code-location-override.png"
-width={700}
-height={500}
+alt="Managing user teams and roles for a user in Dagster Cloud"
+src="/images/dagster-cloud/user-token-management/manage-new-user-permissions.png"
+width={1792}
+height={1474}
 />
 
-#### Branch deployment permissions
+### Adding users to teams
 
-Granting access to Branch Deployments gives users access to all Branch Deployments for all the code locations they have access to. Consider the following example:
+<Note>User teams are a Dagster Cloud Enterprise feature.</Note>
 
-<!-- ![User roles for prod, staging, and all Branch Deployments](/images/dagster-cloud/user-token-management/branch-deployment-example.png) -->
+Using the **Teams** field, you can add users to one or more teams. This is useful for centralizing permission sets for different types of users. Refer to the [Managing user teams](/dagster-cloud/account/managing-users/managing-user-teams) guide for more info about creating and managing teams.
 
 <Image
-alt="User roles for prod, staging, and all Branch Deployments"
-src="/images/dagster-cloud/user-token-management/branch-deployment-example.png"
-width={992}
-height={636}
+alt="Adding a user to a team in the Dagster Cloud UI"
+src="/images/dagster-cloud/user-token-management/add-user-to-teams.png"
+width={1742}
+height={376}
 />
 
-In this case, the user would have **Editor** access to Branch Deployments for all code locations in the `prod` deployment, but not in `staging`. This is because they're a **Viewer** for the code locations in the `prod` deployment, but they don't have access to the `staging` deployment or any of its code locations.
-
----
-
-## Adding users
-
-<Note>
-  <strong>Organization Admin</strong> or <strong>Admin</strong> permissions are
-  required to add users in Dagster Cloud.
-</Note>
+**Note**: When determining a user's level of access, Dagster Cloud will use the **most permissive** role assigned to the user between all of their team memberships and any individual role grants. Refer to the [Managing user roles and permissions](/dagster-cloud/account/managing-users/managing-user-roles-permissions#applying-role-overrides) guide for more info.
```

Comment:
i think this the clearest way to explain it. There's some nuance/details with code location grants that might just complicate things to mention here? Since your level of access can also vary across different code locations